### DISCUSSION TITLE
api-standards repository missing?

### DISCUSSION THREAD

**Discussion by afink170** [trimble-oss/contribute#11] [General]
My team has been asked to conform the the [Trimble HTTP API Standards](https://api-standards.trimble-pnp.com/api-standard/http) for our APIs, and I'd like to look at the versions for the spec, but when I navigate to the link provided in the standards page (https://github.com/trimble-oss/api-standards), it directs me to a page that doesn't exist.

Is the actual spec (the GitHub repo or project) for the the api-standards published anywhere? I see links to it at https://github.com/trimble-oss#api-standards, but they all link back to the same page that doesn't exist.

**Comment by coliff** [comment]
~~Hi. I'm not an Admin of that repo so can't add you myself... but hopefully @jeffdoolittle can get you added.~~
~~It could be helpful if this repo was available to everyone who is part of the trimble-oss org.~~

**Comment by coliff** [comment]
Update: I saw that you aren't part of the trimble-oss org yet, so that's probably why you can't see it yet. To get access there's a few steps to go through:

1. Go to Okta homepage: https://trimble.okta.com/app/UserHome and go to 'Add apps' and add GitHub Enterprise app. It should be added to your account immediately.
2. To get access to trimble-oss Org on GitHub you need to send an email using this template: https://trimble-oss.github.io/contribute/#request-to-join - copy and paste, edit the fields and send.

Important: make sure you include your **GitHub username** (not email address).
You should get access within about 24 hours usually. Once you're added you'll have the Trimble org appearing on your GitHub profile like you can see here: `https://github.com/martynfisher`

**Comment by afink170** [comment]
Thanks, that was it! I was added to the Trimble org, and now I can access the page.

**Reply by devonsparks** [reply]
And if you have questions about the standard, or suggestions for improvement, please don't hesitate to post in the repo's Discussions area 😄: https://github.com/trimble-oss/api-standards/discussions
